src/ovp_pipeline/chat_fileops.py

```
@@ -0,0 +1,857 @@
+    end = text.find("\n---", 4)
+    if end < 0:
+        return "", text
+    raw = text[4:end]
+    body_start = end + len("\n---")
+    if body_start < len(text) and text[body_start] == "\n":
+        body_start += 1
+    body = text[body_start:]
+    return raw, body
+
+
+def _read_top_level_value(raw: str, key: str) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_read_top_level_value`

(ANN401)